src/moderation.ts

```diff
@@ -186,6 +192,56 @@ export class Moderation {
     );
   }
 
+  /**
+   * Appeal against the moderation decision
+   * @param {string} text
+   * @param {string} entityID
+   * @param {string} entityType
+   * @param {Array<string>} attachments
+   * @param {string} channelCID
+   */
+  async appeal(
```

Comment:
@varbhat would it be possible to have a single object argument with these positional arguments as keys inside? It is a better DX to avoid so many positional params I believe.